scripts/system/create/modules/brokenUrlReport.js

```diff
@@ -0,0 +1,390 @@
+//
+//  brokenUrlReport.js
+//
+//  Created by Alezia Kurdis on February 22, 2021.
+//  Copyright 2021 Vircadia contributors.
+//
+//  This script add reports functionalities to the Create Application.
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+//
+
+var brokenUrlReportHttpRequest;
+var brokenUrlReportUrlList = [];
+var brokenUrlReportInvalideUrlList = [];
+var brokenUrlReportProcessedUrlNo = 0;
+var brokenUrlReportUrlEntry;
+var brokenUrlReportMessageBox;
+var brokenUrlReportOverlayWebWindow;
+var BROKEN_URL_REPORT_YES_BUTTON = 0x4000;
+var BROKEN_URL_REPORT_NO_BUTTON = 0x10000;
+var MAX_URL_BEFORE_WARNING_FOR_LONG_PROCESS = 20;
+
+function brokenUrlReportRequestUrlValidityCheck(no) {
+    brokenUrlReportHttpRequest = new XMLHttpRequest();
+    brokenUrlReportHttpRequest.requestComplete.connect(brokenUrlReportGetResponseStatus);
+    brokenUrlReportHttpRequest.open("GET", brokenUrlReportUrlList[no].url);
+    brokenUrlReportHttpRequest.send();
+}
+
+function brokenUrlReportGetResponseStatus() {
+    if (brokenUrlReportHttpRequest.status === 0 || brokenUrlReportHttpRequest.status > 299) {
+        if (brokenUrlReportHttpRequest.status === 0) {
+            brokenUrlReportUrlList[brokenUrlReportProcessedUrlNo].validity = "0 - URL not well-formed";
+        } else {
+            brokenUrlReportUrlList[brokenUrlReportProcessedUrlNo].validity = brokenUrlReportHttpRequest.status + " - " + brokenUrlReportHttpRequest.statusText;
+        }
+        brokenUrlReportInvalideUrlList.push(brokenUrlReportUrlList[brokenUrlReportProcessedUrlNo]);
+    }
+    brokenUrlReportHttpRequest.requestComplete.disconnect(brokenUrlReportGetResponseStatus);
+    brokenUrlReportHttpRequest = null;
+    brokenUrlReportProcessedUrlNo = brokenUrlReportProcessedUrlNo + 1;
+    if (brokenUrlReportProcessedUrlNo === brokenUrlReportUrlList.length) {
+        brokenUrlReportGenerateFormatedReport(brokenUrlReportInvalideUrlList);
+        brokenUrlReportUrlList = [];
+        brokenUrlReportInvalideUrlList = [];
+        brokenUrlReportProcessedUrlNo = 0;
+    } else {
+        brokenUrlReportRequestUrlValidityCheck(brokenUrlReportProcessedUrlNo);
+    }
+}
+
+function brokenUrlReportGenerateFormatedReport(brokenUrlReportInvalideUrlList) {
+    var brokenUrlReportContent = "";
+    if (brokenUrlReportInvalideUrlList.length === 0) {
+        brokenUrlReportContent = "<h1>Broken Url Report<br><br>" + brokenUrlReportUrlList.length + " URL tested.<br><font style='color:#00FF00;'>NO ISSUE HAS BEEN FOUND.</font></h1><br><br><br>";
```

Comment:
Use CSS instead of the multiple `<br>`s at the end of the string?

Also, instead of "NO ISSUE HAS BEEN FOUND", say "NO ISSUES HAVE BEEN FOUND"?

Reply:
I changed for: "NO ISSUES HAVE BEEN FOUND"
 Plural makes sense in this case.
 
 For the <br> using CSS will just take more code to do the same thing. 
 and this is for this specific report only.